package.json

```diff
@@ -79,14 +79,20 @@
     "react-dom": "^0.14.9 || ^15.3.0 || ^16.0.0-rc || ^16.0"
   },
   "dependencies": {
-    "babel-plugin-module-resolver": "^3.0.0",
+    "browserify-global-shim": "^1.0.3",
+    "cssnano": "^3.10.0",
     "draft-js": "^0.10.4",
     "draft-js-export-html": "github:plotly/draft-js-export-html",
     "draft-js-import-html": "^1.2.1",
     "draft-js-utils": "^1.2.0",
+    "es6-shim": "^0.35.3",
     "fast-isnumeric": "^1.1.1",
+    "mdi-react": "^2.1.19",
     "plotly-icons": "^1.0.1",
```

Comment:
@aulneau you're going to add the additional mdi-react icons we need to plotly-icons?